Sources/MapboxDirections/TollCollection.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+/**
+ `TollCollection` describes corresponding object on the route.
+ */
+public struct TollCollection: Codable, Equatable {
+
+    public enum CollectionType: String, Codable {
+        case booth = "toll_booth"
+        case gantry = "toll_gantry"
+    }
+
+    let type: CollectionType
```

Comment:
This property is internal, so clients such as the navigation SDK don’t have access to it.